app/actors/sufia/create_with_files_actor.rb

```diff
@@ -46,10 +46,32 @@ def uploaded_files
       def mark_inactive
         return true unless Flipflop.enable_mediated_deposit?
         curation_concern.state = inactive_uri
+        send_inactive_message
       end
 
       def inactive_uri
         ::RDF::URI('http://fedora.info/definitions/1/0/access/ObjState#inactive')
       end
+
+      def send_inactive_message
+        return true if receiving_users == []
+        receiving_users.each do |admin|
```

Comment:
Admins are receivers at the moment, but the intent is to expand this to more types of users soon. Suggest renaming this to `recipient`.